src/Sylius/Bundle/CoreBundle/EventListener/CartBlamerListener.php

```diff
@@ -56,6 +57,16 @@ public function onInteractiveLogin(InteractiveLoginEvent $interactiveLoginEvent)
         $this->blame($user);
     }
 
+    public function onJWTAuthenticatedLogin(JWTAuthenticatedEvent $JWTAuthenticatedEvent): void
```

Comment:
```suggestion
    public function onJWTAuthenticatedLogin(JWTAuthenticatedEvent $event): void
```

Reply:
I would stay with JWTAuthenticatedEvent to be consistent with rest of functions in this class

Reply:
I also vote for `event` here